webapp/src/js/controllers/inbox.js

```diff
@@ -417,7 +418,7 @@ const moment = require('moment');
       },
       // tour
       {
-        required: (settings, user) => !user.known,
+        required: (settings, user) => !user.known && ctrl.tours.length > 0,
```

Comment:
I think there might be a race condition here - `ctrl.tours` is initialised to `[]` and then we fetch the tours from the database. If the filter is run before the tours array is populated then it'll think none are available.

I think you can change it so around line 430 in the `$q.all` it calls `initTours` as well instead of it being called on line 256 and that way we know tours is set before running this code.